docs/ui/cloud.md

```diff
@@ -93,6 +94,10 @@ Role-based access control (RBAC) functionality in Prefect Cloud enables you to a
 
 See the [role-based access controls (RBAC)](/ui/roles/) documentation for more information about managing user roles in a Prefect Cloud organization.
 
+## Single Sign-on (SSO) <span class="badge orgs"></span> <span class="badge enterprise"></span>
+
+Prefect Cloud [Enterprise plans](https://www.prefect.io/pricing) offer [single sign-on (SSO)](/ui/sso/) authentication integration with your team’s identity provider. SSO integration can bet set up with identity providers that support OIDC and SAML.
```

Comment:
```suggestion
Prefect Cloud [Organization plans](https://www.prefect.io/pricing) and above offer [single sign-on (SSO)](/ui/sso/) authentication integration with your team’s identity provider. SSO integration can bet set up with identity providers that support OIDC and SAML.
```

I think this is more correct since we offer this on all self-serve tiers